.github/workflows/build_and_test.yml

```diff
@@ -79,8 +79,10 @@ jobs:
     - name: Install docs dependencies
       if:  (matrix.os == 'ubuntu-latest') && (github.event_name == 'pull_request' || github.event_name == 'schedule' )
       run: |
-        cd docs && bash ./install.sh
-        for w in `find wheelhouse/ -type f -name "*.whl"` ; do poetry run pip install $w ; done
+        bash ./docs/install.sh
+        cd docs
+        for w in `find ../wheelhouse/ -type f -name "*.whl"` ; do poetry run pip install $w ; done
```

Comment:
Seems the reason this wasn't working it that we couldn't find the `wheelhouse` directory after "cd"ing into docs